test/Sentry.Unity.Tests/EmbeddedResourcesTests.cs

```diff
@@ -0,0 +1,18 @@
+using System.Linq;
+using System.Reflection;
+using NUnit.Framework;
+
+namespace Sentry.Unity.Tests
+{
+    public sealed class EmbeddedResourcesTests
+    {
+        [Test]
+        public void LinkXml_Embedded()
+        {
+            var resourceNames = Assembly.GetAssembly(typeof(SentryUnity)).GetManifestResourceNames();
```

Comment:
I'm not sure if this is the proper way to get to the Assembly so any feedback would be greatly appreciated.